pkg/network/tracer/tracer.go

```diff
@@ -244,16 +243,16 @@ func newConntracker(cfg *config.Config, bpfTelemetry *nettelemetry.EBPFTelemetry
 	var c netlink.Conntracker
 	var err error
 
-	// try creating ebpf conntracker 3 times in case the module is not loaded on the host yet
-	err = retry.Do(
-		func() error {
-			c, err = NewEBPFConntracker(cfg, bpfTelemetry)
-			return err
-		},
-		retry.Attempts(3),
-		retry.Delay(1*time.Second),
-	)
-	if err == nil {
+	ns, err := cfg.GetRootNetNs()
+	if err != nil {
+		log.Warnf("error fetching root net namespace %s", err)
```

Comment:
You could add that we're not going to try loading up the conntrack module in the log message here.